src/modules/data.js

```diff
@@ -3,13 +3,18 @@ import {deepEqual} from 'fast-equals';
 
 import './data.css';
 export const CHANGE_EXPR = 'CHANGE_EXPR';
+export const RESET_HIGHLIGHT = 'RESET_HIGHLIGHT';
 
 type Action = {
```

Comment:
If you need a second action in this module, I would rename this type to `ChangeAction`.